libs/ui/src/themes/make_theme.ts

```diff
@@ -224,6 +228,7 @@ export const colorThemes: Record<ColorMode, ColorTheme> = {
     primary: TouchscreenPalette.Purple80,
     danger: TouchscreenPalette.Red80,
     warningAccent: TouchscreenPalette.Gray90,
+    warningContainer: TouchscreenPalette.Orange20,
```

Comment:
Why did you decide to add `warningContainer` instead of just change `warningAccent` and special casing _all_ use cases to add a black border?

Reply:
One option to look into might be using the font awesome "regular" style with a background color instead of the "solid" style

Reply:
Yeah I thought about this but ran into 2 (surmountable) hurdles in my limited time:

1. Most of the regular style icons (including exclamation triangle) require Font Awesome Pro. I actually bought a subscription for the team accordingly while investigating this issue. We'd just need to take some time to configure our app with the extra icons. Also, if we're using Font Awesome Pro, I think we should just use their provided duotone icons when possible.
2. Applying a background color to a regular style icon doesn't work out of the box but I'm sure can be solved for the different categories of shapes.

<img width="250" height="250" alt="shape" src="https://github.com/user-attachments/assets/c747d938-ec01-4239-8cb1-ce32e576b7f2" />

Reply:
Landed on this for now: https://github.com/votingworks/vxsuite/pull/7137/commits/dd58f6974723d8c66878fc83c7895a1840dd90ac

Took care to make sure that we have no existing cases that'll trigger the assertion check - found just one case that would trigger: https://github.com/votingworks/vxsuite/pull/7137/commits/ba05d5a7acfe4050d611429fd7b0838220c66b23

Reply:
Ah ok yeah I see the wrinkles. This seems like a good enough solution for now. Thanks for sharing the additional context!